src-built-in/components/appLauncher2/src/components/Content.jsx

```diff
@@ -0,0 +1,39 @@
+import React from  'react'
+import AppDefinition from './AppDefinition'
+import FilterSort from './FilterSort'
+
+export default class Content extends React.Component {
+
+	constructor(props) {
+		super(props)
+		this.state = {
+			filter: {
+				text: ''
+			}
+		}
+	}
+
+	filterApps() {
+		const apps = this.props.activeFolder.appDefinitions
```

Comment:
Once you get to making this filter work, you'll want your filterSort component to set a value on the store; this component will listen for changes on that field. When the value changes, this component renders with the newly filtered list.